client/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ add_executable(
 target_compile_definitions(
   ${IRODS_EXECUTABLE_NAME}
   PRIVATE
-  #${IRODS_COMPILE_DEFINITIONS}
+  #${IRODS_COMPILE_DEFINITIONS} # TODO Is this needed?
```

Comment:
so we're still unsure on this?

Reply:
Based on the macros defined by this variable, I think these should always be set. Not all of them are necessary for clients, but the majority are.

See https://github.com/irods/irods/blob/1205fa3afd8de1e96e0e412184c7b30ae0b07c31/CMakeLists.txt#L216-L239 for the definitions.

Reply:
Marking as resolved.